### DISCUSSION TITLE
Optimization or No ?

### DISCUSSION THREAD

**Discussion by kawaii-ghost** [kawaii-ghost/x64-linux-asm#1] [Polls]
Jadi sebenernya sudah saya jelasin beberapa optimisasi seperti kenapa pakai register 32 bit (REX prefix) dan xor.

Sebenernya saya tidak mempermasalahkan kode-kode yang nggak optimal. Buktinya di bagian perkenalan saya nggak langsung maen optimisasi tapi jelasin dulu baru kedepannya optimisasi.

**Comment by alviroiskandar** [comment]
Ya lu kasih contoh yang best practice gpp, setidak na orang2 tau gmn code
yg baik dan optimal. Tapi ga usah di bahas sampai ke opcode segala ampe
REX prefix dan ukuran code na. Ga terlalu penting di awal. Tapi nunjukin
best practice, gimana nulis code yg baik itu perlu diajarkan sejak dini

Singkat na, lu ngajarin:
"Ini adalah cara yang terbaik, pakailah cara ini"

ga usah peduli alasan na knp

simple

**Reply by kawaii-ghost** [reply]
Shap

**Comment by dimasma0305** [comment]
Oke, saya bikin yang versi optimisasinya, tapi pasti ada yang kurang sih, jadi kalau ada tambahan bisa langsung di commit aja.

**Reply by dimasma0305** [reply]
wah agak telat, sudah di optimisasi ternyata. Okelah ta kasi gambar aja yang example.
